core/src/main/java/com/linecorp/armeria/common/thrift/ThriftCallbackCompletableFuture.java

```diff
@@ -0,0 +1,22 @@
+package com.linecorp.armeria.common.thrift;
```

Comment:
- I think it's fine as it is. What makes you think there's a better place?
- Missing copyright header btw :-)